R/demultiplex.R

```diff
@@ -198,17 +198,17 @@ demultiplex_reads <- function(sample_manifest,
       reads_2 = character(),
       n = integer()
     )
-
+  
   future_rng_opt <- getOption('future.rng.onMisuse')
   options(future.rng.onMisuse = "ignore")
   on.exit(options(future.rng.onMisuse = future_rng_opt))
-
+  
   workers <- list(
-    future::makeClusterPSOCK(workers = 1),
+    future::makeClusterPSOCK(workers = floor(future::availableCores()/2)),
```

Comment:
Please revert, we want precisely two workers here, 1 for forward read and one for reverse. More workers is not helpful.